pkg/lib/cockpit-connect-ssh.tsx

```diff
@@ -23,6 +23,8 @@
 
 import React, { useState } from 'react';
 
+import 'page.scss';
```

Comment:
Libraries shouldn't do that. The intention is that each page includes `page.scss` *once* at the very top level. Including it here again will lead to duplicatino.

Reply:
Right. Should I put it in `lib/playground/remote.tsx`?